### ISSUE TITLE
Szenenzahl "Götz von Berlichingen" (11g1d.0)

### ISSUE THREAD

**Issue by BenjaminKrautter** [quadrama/Corpus#8]
Welche Version von Goethes "Götz" nutzt ihr im Aufsatz "Poetologischer Anspruch und dramatische Wirklichkeit"? Im Aufsatz steht, dass die Szenenzahl pro Akt zwischen 5 und 22 variiert. Die aktuelle Version im Korpus `gdc:11g1d.0` hat jedoch in jedem Akt genau eine Szene. Habe gerade auch in Textgrid und einer Buchausgabe geschaut. Szenen werden nicht explizit genannt, es gibt jedoch sehr viele Auf- und Abtrittte und Ortswechsel. Wurde das damals implementiert?

**Comment by nilsreiter** [comment]
Das ist glaube ich übernommen aus dem Edinburgh-Aufsatz, ist also uralt. Und dann müsste es TextGrid sein.

**Comment by nilsreiter** [comment]
Ich finde die Szenenzahlen 5, 10, 22, 5, 14 in den Tabellen

**Comment by nilsreiter** [comment]
@marcuswilland weißt Du das noch?

**Comment by marcuswilland** [comment]
Nein, ich finde leider auch keine Hinweise in den Arbeitsnotizen von damals. Das ist ziemlich sicher ein Fehler. Good spotting, @BenjaminKrautter. Hör jetzt auf, unsere Fehler zu finden 👎 
Nein, ernsthaft. Ich gehe davon aus, dass da beim Berechnen der vielen Standardabweichungen ein Fehler passiert ist oder beim Ablesen der richtigen Berechnung der Fehler passiert ist. Die 5/22-Varianz ist sicherlich die eines anderen Stücks. Blöd, dass ich das damals nicht geprüft habe. 
Blöder noch, wenn das ein systematischer Fehler wäre. 
Am blödesten, dass wir die Daten in zwei Aufsätzen verwendet haben

**Comment by marcuswilland** [comment]
Es gibt im DB-Ordner "....QuaDrama\Vorträge, Tagungen, Publikationen\2016_anglo-germanencounters_edinburgh\Daten" vllt. noch Hinweise dazu?

**Comment by nilsreiter** [comment]
Der R-Code liegt bei mir auf dem Laptop, da habe ich noch diese Zeilen gefunden:
```
thisId <- "tg:11g1d.0"
scenes.tbl.agg[nrow(scenes.tbl.agg)+1,] <- c(thisId,1,5)
scenes.tbl.agg[nrow(scenes.tbl.agg)+1,] <- c(thisId,2,10)
scenes.tbl.agg[nrow(scenes.tbl.agg)+1,] <- c(thisId,3,22)
scenes.tbl.agg[nrow(scenes.tbl.agg)+1,] <- c(thisId,4,5)
scenes.tbl.agg[nrow(scenes.tbl.agg)+1,] <- c(thisId,5,14)
```

Wir haben die Einträge explizit bei Götz nachgetragen. Das hätten wir ja nicht ohne Grund gemacht. Vielleicht basierend auf einer Druckfassung?

**Comment by nilsreiter** [comment]
Das basiert auf den Zwischenüberschriften, die z.B. bei [gutenberg.spiegel.de](http://gutenberg.spiegel.de/buch/gotz-von-berlichingen-mit-der-eisernen-hand-3672/3) fett ausgezeichnet sind.

**Comment by nilsreiter** [comment]
... und damit angeblich auf der Reclam-Ausgabe von 1993.

**Comment by marcuswilland** [comment]
Mein Reclam-Götz steht in Heidelberg, aber die anderen Reclams machen das auch so:
https://www.reclam.de/data/media/978-3-15-019153-8.pdf
Meine Weimarer Ausgabe steht leider noch bei den Eltern

**Comment by nilsreiter** [comment]
Dann haben wir's ja geklärt und es war kein Fehler! :)

**Comment by marcuswilland** [comment]
Ja, das beruhigt mich sehr

**Comment by BenjaminKrautter** [comment]
Ja, genau. Hatte in der Zwischenzeit auch nochmal in einer HKA nachgeschaut, es aber hier nicht mehr reingeschrieben. Der Fehler liegt in unserer gdc Fassung, nicht in der ergänzten von eurem Aufsatz. Sollte man entsprechend auch noch fixen (außer wir nehmen übernehmen sowieso die neuesten gerdracor Daten).

Die Berechnungen in unserem Litlab Aufsatz sind also betroffen. Da sollte es allerdings keinen wirklichen Einfluss haben.

**Comment by nilsreiter** [comment]
Ist es denn in den neueren GDC-Daten gefixt?

**Comment by BenjaminKrautter** [comment]
Wenn ich das richtig sehe, ja.

[https://github.com/dracor-org/gerdracor/blob/master/tei/goethe-goetz-von-berlichingen-mit-der-eisernen-hand.xml](url)

**Comment by pagelj** [comment]
Es gibt zumindest in der Version 55 Szenen
